backend/de.metas.acct.base/src/main/sql/postgresql/system/43-de.metas.acct/5588320_sys_gh11094_Products_Winthout_Initial_Cost_Function.sql

```diff
@@ -0,0 +1,80 @@
+DROP FUNCTION IF EXISTS ProductsWithoutInitialCost
+(p_D_Client_ID NUMERIC,
+ p_ad_org_id NUMERIC);
+
+CREATE OR REPLACE FUNCTION ProductsWithoutInitialCost(
+    p_AD_Client_Id NUMERIC,
+    p_AD_Org_Id NUMERIC)
+    RETURNS table
+            (
+                AD_Org_ID    numeric,
+                OrgValue     text,
+                OrgName      text,
+                M_Product_ID numeric,
+                ProductValue text,
+                ProductName  text
+            )
+AS
+$$
+WITH element AS
+         (
+             SELECT el.m_costelement_id
+
+             FROM c_acctschema s
+                      JOIN ad_clientinfo ci on ci.ad_client_id = p_AD_Client_Id
+                 AND s.c_acctschema_id = ci.c_acctschema1_id
+                      JOIN m_costelement el on s.costingmethod = el.costingmethod
```

Comment:
I would introduce the p_M_CostElement_ID parameter.
Is there any reason to not do it?

Reply:
> I would introduce the p_M_CostElement_ID parameter.
> Is there any reason to not do it?

Added it and made it mandatory. The default value will be the one with the same costing method as the one form the schema in the client info.